### DISCUSSION TITLE
Importing data to droplet installation

### DISCUSSION THREAD

**Discussion by nunofrcosta0411** [inventree/InvenTree#6345] [Q&A] [answered]
I have setup a Docker installation, running InvenTree 0.13.0, in our own physical server for testing. Everything is running fine and we are doing daily backups that I copy to a local folder using SFTP.I have also exported the data to a JSON file as suggested...

Meanwhile, we want to access data from anywhere and have used the DigitalOcean droplet running version 0.11.0.

How can I import the JSON data file or do a DB restore from the backup in the droplet version? I have been trying to connect PgAdmin to the droplet database from [here](https://github.com/joshborup/PostgreSQL-digital-ocean-remote-access-via-pgadmin4/blob/master/README.md) without success!

**Comment by matmair** [comment]
You can use the cli on droplet installations with `inventree run invoke` - importing the json with `inventree run invoke import-records`.
Setting up pgadmin access to the droplets db server can open you up to hacking of you postgres instance, I would not recommend it if you are not a seasoned linux admin. By default the droplet ships with pretty tight firewall rules that disallow it.

**Reply by nunofrcosta0411** [reply, marked as answer]
Thank for the reply. Still no luck...

After copying the JSON files to **/opt/inventree**, ran `inventree run invoke import-records` as you suggested and got the error:

> Importing database records from '/opt/inventree/data.json'
> python3: error while loading shared libraries: libpython3.10.so.1.0: cannot open shared object file: No such file or directory

Also tried to send the 2 backup files I got from the Docker installation to **/opt/inventree/data/backup** and do `invoke restore`. First it complained couldn't find 'django'. Then after installing it, it complained about a `moneyd `module...

Can't seem to put it to work in DigitalOcean after such an easy installation in Docker!

**Comment by nunofrcosta0411** [comment]
Managed to put it to work in DigitalOcean using a Docker droplet instead and following the Docker installation instructions.

For those of you who have been struggling with the same problems, here is what I did:

1. On the physical server running Docker just issued `docker compose run inventree-server invoke backup`. 
2. Using WinSCP transfered the 2 files locally and then sent them to DigitalOcean's droplet into **/opt/inventree/data/backup** (the directory where I installed InvenTree).
3. Open a CLI in DigitalOcean's droplet and issue `docker compose run inventree-server invoke restore`

As a side note, I am using a Basic, Premium AMD, 1vCPU, 2Gb RAM, 25Gb SSD and 2Tb transfer rate architecture. The most basic (and cheapest) was struggling...

Cheers.

**Reply by matmair** [reply]
Thank you for reporting back!
